mrLoadRet/Plugin/GLM_v2/newGlmAnalysis/hrfDoubleGamma.m

```diff
@@ -16,6 +16,12 @@
   sampleDelay=sampleDuration/2;
 end
 
+if sampleDuration == 1000

```

Comment:
It seems that this issue comes from having stim times in milliseconds instead of seconds, and it looks like this fix is only going to work in a very specific case where the duration has been set to 1000 ms (instead of 1s). This fix wouldn't work if it has been set to 2000ms.
I think the question here is why is the sample duration field was set to 1000 to start with? So I suspect that the fix should be somewhere else and there isn't really a bug here.
This change is unlikely to affect anything for other users because no one will ever actually use a duration of 1000s, but it's not very general.